### PR TITLE
Add CvTONNX Config

### PR DESCRIPTION
**Pull request by rishabbala**
# What does this PR do?
Added changes for CvT model

Fixes part of #555

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

@michaelbenayoun @fxmarty

### CODE REVIEW

**Comment by fxmarty** [comment]
Hi, thank you for the addition! Tests indeed pass, except the style one. Could you run `make style`?

File: optimum/exporters/onnx/model_configs.py [hunk 1]
```diff
@@ -550,6 +550,11 @@ def inputs(self) -> Dict[str, Dict[int, str]]:
         return {"pixel_values": {0: "batch_size", 1: "num_channels", 2: "height", 3: "width"}}
 
 
+class CvTOnnxConfig(ViTOnnxConfig):
+    DEFAULT_ONNX_OPSET = 13
+    ATOL_FOR_VALIDATION = 1e-2
```
Comment (michaelbenayoun) [inline optimum/exporters/onnx/model_configs.py:555]: This seems pretty high tbh, do you have any idea which op makes it fail?

**Comment by rishabbala** [comment]
@fxmarty  Done

File: optimum/exporters/onnx/model_configs.py (hunk 1 above)
Comment (michaelbenayoun) [reply optimum/exporters/onnx/model_configs.py:555]: I am talking about the tolerance, we might have a hidden bug if we need `1e-2`, or we might not, but just wanted to raise the potential issue here.

**Review by fxmarty — APPROVED** [review]
LGTM, thank you for the addition!

@rishabbala Maybe before merging, could you try to run the export on a larger CvT model and see what the diff is versus pytorch in the validation?

**Comment by rishabbala** [comment]
The diff for cvt-13 is 3.5e-3, for cvt-13-384-22k it is 6e-3 and for cvt-21 it is 2.7e-3. Should I reduce the tolerance?

**Comment by fxmarty** [comment]
Sounds fine to me, thanks!

**Pull request merged**